dts/bindings/pinctrl/pincfg-node-group.yaml

```diff
@@ -1,165 +0,0 @@
-# Copyright (c) 2020, Linaro Limited
-# SPDX-License-Identifier: Apache-2.0
-
-description: |
-  Generic pin configuration schema.
-
-  This file is equivalent to pincfg-node.yaml but inserts properties at a
-  grandchildren level, useful for group based representation.
```

Comment:
Don't you think we should keep this here, for out of tree users who are depending on it?

Not blocking on this one since I no longer maintain bindings, but I would strongly suggest it.

Reply:
> I’d be surprised if there are pinctrl oot users, since it’s a de-facto requirement for any platform.

There are out of tree socs and even out-of-tree architectures. These are first class citizens in Zephyr. I understand you want to make changes quickly but I would not agree that the new treewide change policy is "unnecessary" and neither did the rest of the TSC.